javascript/core/src/verbs/util/boolean-logic.ts

```diff
@@ -156,3 +161,20 @@ export function nand(comparisons: (1 | 0 | null)[]): 1 | 0 | null {
 	}
 	return null
 }
+
+/**
+ * Logical XNOR
+ * @param comparisons
+ * @returns
+ */
+ export function xnor(comparisons: (1 | 0 | null)[]): 1 | 0 | null {
```

Comment:
Please double-check that the code comments on each of these functions matches with our latest and most accurate descriptions of the logic.